docs/src/__examples__/TileGroup/DEFAULT.tsx

```diff
@@ -28,9 +28,10 @@ export default {
       />
     </TileGroup>
   ),
-  info: {
-    title: "Default tile group",
-    description:
-      "Tile groups can contain many things. To keep the proper styling, don't include any DOM elements other than tiles.",
-  },
+  exampleKnobs: [
+    {
+      component: "TileGroup",
+      knobs: [{ name: "dataTest", type: "text", defaultValue: "" }],
+    },
+  ],
```

Comment:
I don't think there's any point in having only `dataTest`, I'd remove knobs altogether here.